tensorflow/python/ops/math_ops.py

```diff
@@ -144,6 +144,15 @@ def argmax_v2(input, axis=None, output_type=dtypes.int64, name=None):
 
   Note that in case of ties the identity of the return value is not guaranteed.
 
+  For example:
+  ```python
+  A=tf.constant([2,20,30,3,6]) # Constant 1-D Tensor
+  tf.math.argmax(A) # output 2 as index 2 (A[2]) is maximum in tensor A
+  B=tf.constant([[2,20,30,3,6],[3,11,16,1,8],[14,45,23,5,27]])
+  tf.math.argmax(B,0) # [2, 2, 0, 2, 2]
+  tf.math.argmax(B,1) # [2, 2, 1]
+   ```
```

Comment:
nit: you have an extra space here, please remove one.

Reply:
Thanks. Removed the space as you suggested. Thanks for your help.

Reply:
@tomhennigan Is there anything missing from my side? Thanks!